frontend/src/main/scala/bloop/engine/Build.scala

```diff
@@ -1,13 +1,27 @@
 package bloop.engine
 
 import bloop.Project
-import bloop.io.AbsolutePath
+import bloop.io.{AbsolutePath, DirChecksum}
+
+final case class Build private (origin: AbsolutePath,
+                                projects: List[Project],
+                                originChecksum: DirChecksum) {
 
-final case class Build(origin: AbsolutePath, projects: List[Project]) {
   private val stringToProjects: Map[String, Project] = projects.map(p => p.name -> p).toMap
   private[bloop] val dags: List[Dag[Project]] = Dag.fromMap(stringToProjects)
 
   def getProjectFor(name: String): Option[Project] = stringToProjects.get(name)
   def getDagFor(project: Project): Dag[Project] =
     Dag.dagFor(dags, project).getOrElse(sys.error(s"Project $project does not have a DAG!"))
+
+  /** Has this build definition changed since it was loaded? */
+  def changed(): DirChecksum.DirStatus = originChecksum.changed()
```

Comment:
No parens for `changed` in the method definition?

Reply:
I've put the parentheses because `changed` isn't pure